docs/theme/src/cookbook/markdown/demo.md

```diff
@@ -72,11 +72,11 @@ This is another paragraph.
 
 ## Line Break
 
-I would like to line break at  
+I would like to line break at
```

Comment:
Just realize your pr breaks en markdown demo.

Reply:
oops I will fix that. truly sorry that I didn't find this out

Reply:
Well since you have already fixed this, I shall use that as a reminder to double check next time